### PR TITLE
add end point get animal models

### PR DESCRIPTION
**Pull request by AlexeyDerepa**

### CODE REVIEW

File: Backend/AnimalRescue/AnimalRescue.Contracts/Services/IAnimalService.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using AnimalRescue.Models.DTO.Models;
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace AnimalRescue.Contracts.Services
+{
+    public interface IAnimalService
+    {
+        Task<List<AnimalModel>> GetAnimalAsync(int currentPage = 1, int pageSize = 10);
```
Comment (AlistratovAlex) [inline Backend/AnimalRescue/AnimalRescue.Contracts/Services/IAnimalService.cs]: GetAnimalAsync -> GetAnimalsAsync

File: Backend/AnimalRescue/AnimalRescue.DataAccess.Contracts/Interfaces/IAnimalRepository.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using AnimalRescue.Models.DTO.Models;
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace AnimalRescue.DataAccess.Contracts.Interfaces
+{
+    public interface IAnimalRepository
+    {
+        Task<List<AnimalModel>> GetAnimalAsync(int currentPage = 1, int pageSize = 10);
```
Comment (AlistratovAlex) [inline Backend/AnimalRescue/AnimalRescue.DataAccess.Contracts/Interfaces/IAnimalRepository.cs]: GetAnimalAsync->GetAnimalsAsync

File: Backend/AnimalRescue/AnimalRescue.DataAccess.Mongodb/Models/Animal.cs [hunk 3]
```diff
@@ -11,9 +11,10 @@ public class Animal : BaseItem
         public string Gender { get; set; }
         public string Description { get; set; }
         public int Age { get; set; }
+        public DateTimeOffset? ModifiedBy { get; set; }
         public DateTimeOffset DateOfFound { get; set; }
```
Comment (AlistratovAlex) [inline Backend/AnimalRescue/AnimalRescue.DataAccess.Mongodb/Models/Animal.cs]: Please change Number data type long to int 
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/integral-numeric-types

Comment (AlistratovAlex) [reply Backend/AnimalRescue/AnimalRescue.DataAccess.Mongodb/Models/Animal.cs]: Also don't forget about BsonDiscriminator and BsonElement attributes

File: Backend/AnimalRescue/AnimalRescue.DataAccess.Contracts/Interfaces/IAnimalRepository.cs (hunk 2 above)
Comment (AlexeyDerepa) [reply Backend/AnimalRescue/AnimalRescue.DataAccess.Contracts/Interfaces/IAnimalRepository.cs]: done

File: Backend/AnimalRescue/AnimalRescue.Contracts/Services/IAnimalService.cs (hunk 1 above)
Comment (AlexeyDerepa) [reply Backend/AnimalRescue/AnimalRescue.Contracts/Services/IAnimalService.cs]: done

**Pull request merged**